check-gh-collab.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+usage () {
+    echo "Usage: check-collab.sh <GITHUB ORG> <GITHUB USERNAME>"
+    exit 1
+}
+
+[[ $# -ne 2 ]] && usage
+
+ORG=$1
+USER=$2
+
+REPOS_PER_PAGE=100
+
+# TODO: figure out a  way to find the last page containing repos
+for i in $(seq 5); do
+    repos+=$(curl --stderr /dev/null \
+                  -H "Authorization: token $(cat ~/.github/oauth2)" \
```

Comment:
You should check up front if this file exists and bail if it does not.

Also check out the `-K` argument to curl to read options from a file (or stdin), which would avoid exposing your token in the output of `ps`.